src/pretalx/schedule/models/room.py

```diff
@@ -18,6 +20,11 @@ class Room(LogMixin, models.Model):
         to="event.Event", on_delete=models.PROTECT, related_name="rooms"
     )
     name = I18nCharField(max_length=100, verbose_name=_("Name"))
+    guid = models.UUIDField(
+        default=uuid4,
+        verbose_name=_("GUID"),
+        help_text=_("Unique room identifier across multiple systems (UUID)."),
```

Comment:
Maybe add a "to help external tools identify the room", "across multiple systems" is a bit generic.